layers_ajax.php

```diff
@@ -72,6 +72,11 @@
       'source' => $layer['cal_layeruser'],
       'color' => $layer['cal_color'],
       'dups' => $layer['cal_dups'],
+// Craig. This looks like a bug. I can't find $layerfullname set anywhere.
+// And the only table with a fullname is webcal_entry_ext_user, which doesn't seem to be called here.
+// Even $fullname doesn't seem to fit.
+// Is it supposed to be maybe $layer['cal_login'] or $layer['cal_layeruser']?
+
       'fullname' => $layerfullname];
```

Comment:
The fullname was supposed to be a convenience for the calling pages.  Not sure if we are still using it anywhere though.  We used to have repeated sections of code that would assign fullname based on first, last and username (mostly handling empty values).